spatialos-sdk/src/worker/alpha.rs

```diff
@@ -0,0 +1,465 @@
+use std::ffi::CString;
+
+use futures::{Async, Future};
+
+use spatialos_sdk_sys::worker::*;
+
+use crate::worker::{
+    internal::utils::cstr_to_string,
+    parameters::ProtocolLoggingParameters,
+};
+
+pub struct Locator {
+    pub(crate) internal: *mut Worker_Alpha_Locator,
```

Comment:
Does `Locator` need a `Drop` impl to free this pointer once it's done?

Reply:
It does indeed, nice catch!